### PR TITLE
CNV-37820: Fix Trans translations

### PR DESCRIPTION
**Pull request by upalatucci**
## 📝 Description

The main cause of errors in console for translation is that `Trans` does not support conditional rendering inside. 
Solution? Split the translation.
We also need to be careful with variables and use double `{ }` like `{{ phase: vmim?.status?.phase }}` and not `{{ vmim?.status?.phase }}`.

## 🎥 Demo

No console errors

### CODE REVIEW

File: src/views/virtualmachines/details/tabs/overview/components/VirtualMachinesOverviewTabDetails/components/MigrationProgressPopover/MigrationProgressPopover.tsx [hunk 1]
```diff
@@ -48,38 +47,30 @@ const MigrationProgressPopover: React.FC<MigrationProgressPopoverProps> = ({ chi
       bodyContent={
         <Stack hasGutter>
           <StackItem>
-            <Trans ns="plugin__kubevirt-plugin">
-              <b>Phase</b> <Icon /> {vmim?.status?.phase}
-            </Trans>
+            <b>{t('Phase')}</b> <Icon /> {{ phase: vmim?.status?.phase }}
```
[the remark this answers is not in the document]
Comment (upalatucci) [reply src/views/virtualmachines/details/tabs/overview/components/VirtualMachinesOverviewTabDetails/components/MigrationProgressPopover/MigrationProgressPopover.tsx:50]: No it was under Trans before. And i changed after :D

File: src/utils/components/ConfirmActionMessage/ConfirmActionMessage.tsx [hunk 2]
```diff
@@ -14,16 +14,18 @@ const ConfirmActionMessage: React.FC<ConfirmActionMessageProps> = ({ action = 'd
   const objNamespace = obj?.metadata?.namespace;
 
   return (
-    <Trans t={t}>
-      Are you sure you want to {action} <strong>{obj?.metadata?.name}</strong>
+    <>
+      <Trans t={t}>
+        Are you sure you want to {{ action }} <strong>{{ name: obj?.metadata?.name }}</strong>
```
[the remark this answers is not in the document]
Comment (upalatucci) [reply src/utils/components/ConfirmActionMessage/ConfirmActionMessage.tsx:19]: It's not just like that. This string is transformed by react but i18n is not detecting that here we are using a variable. 
So react will replace `obj?.metadata?.name` with the actual object name but i18n expect the string `obj?.metadata?.name` to translate the phrase. 

With the `{{ }}` i18n know that obj?.metadata?.name is a variable that can change everytime and is not meant to be translated.

**Review by hstastna — APPROVED** [review]

**Comment by metalice** [comment]
Hey Ugo I know it merged and it wasn't mention in the epic, but

1. we should also update our i18n packages
2. did u check we don't have any import mistakes, and we still have leftovers that import t from useTranslation and not useKubevirtTranslations?
3. We have some utils files/functions that we are sending t as arg instead of importing directly. Please check and remove those args 
4. did u test after this PR? No errors are presented in the console? It needs a deep navigation across the app pages; when looking at the translation JSON, I don't see many new values, and if I remember correctly, the quantity of errors in the console was a lot. Please double-check.

**Comment by upalatucci** [comment]
@metalice 

1. okay
2. yes, there are no other `useTranslation` usage
3. Okay I'll check that. 
4. Yes

**Comment by metalice** [comment]
> @metalice
> 
> 1. okay
> 2. yes, there are no other `useTranslation` usage
> 3. Okay I'll check that.
> 4. Yes

I can see we still have more errors, please go over all Trans components
image

**Pull request merged**